src/lib/relay/QueryRenderers.tsx

```diff
@@ -60,21 +60,24 @@ export const ArtistRenderer: React.SFC<ArtistRendererProps> = ({ render, artistI
 
 interface ConversationRendererProps extends RendererProps {
   conversationID: string
+  cursor: string
+  count: number
 }
 
 export const ConversationRenderer: React.SFC<ConversationRendererProps> = ({ render, conversationID }) => {
   return (
     <QueryRenderer
       environment={environment}
-      query={graphql`
-        query QueryRenderersConversationQuery($conversationID: String!) {
+      query={graphql.experimental`
+        query QueryRenderersConversationQuery($conversationID: String!, $count: Int!, $after: String) {
           me {
-            ...Conversation_me
+            ...Conversation_me @arguments(count: $count, after: $after)
```

Comment:
Why do you want to pass `count` and `after` in from here at all? Why does the root query need to know anything about a pagination container somewhere down the tree?

Reply:
I get the following errors back if I don't include those fields in the Conversation query renderer.
<img width="809" alt="screen shot 2017-09-13 at 2 54 49 pm" src="https://user-images.githubusercontent.com/296775/30395286-a3f710d0-9893-11e7-972a-7f71e8626af8.png">

Reply:
That’s probably because you’re missing the `@argumentDefinitions` [here](https://github.com/artsy/emission/pull/750/files#diff-2bff25af92fc79a4ace9f3098d77d65fR102). Look at the other pagination containers for examples.